tests/unittests/CoreText/CTParagraphStyleTests.mm

```diff
@@ -22,27 +22,27 @@
 
 TEST(CoreText, CTParagraphStyle) {
     CTParagraphStyleRef paragraphStyle = CTParagraphStyleCreate(nullptr, 0);
-    ASSERT_TRUE_MSG(paragraphStyle != nullptr, "FAILED: Could not create paragraph style");
+    EXPECT_TRUE_MSG(paragraphStyle != nullptr, "FAILED: Could not create paragraph style");
 
     CGFloat val = 2.0f;
     CTParagraphStyleSetting settings[1] = { {.spec = kCTParagraphStyleSpecifierTailIndent, .valueSize = sizeof(CGFloat), .value = &val } };
     paragraphStyle = CTParagraphStyleCreate(settings, 1);
-    ASSERT_TRUE_MSG(paragraphStyle != nullptr, "FAILED: Could not create paragraph style");
+    EXPECT_TRUE_MSG(paragraphStyle != nullptr, "FAILED: Could not create paragraph style");
 
     CGFloat bufferFloat;
     bool success =
         CTParagraphStyleGetValueForSpecifier(paragraphStyle, kCTParagraphStyleSpecifierTailIndent, sizeof(CGFloat), &bufferFloat);
-    ASSERT_TRUE_MSG(success == true, "FAILED: Could not found the specifier that was set.");
-    ASSERT_TRUE_MSG(fabs(bufferFloat - val) < 0.00001, "FAILED: Incorrect specifier value received");
+    EXPECT_TRUE_MSG(success == true, "FAILED: Could not found the specifier that was set.");
```

Comment:
These were already here, I just did a replace assert/expect so I could see everything that was failing.